package.json

```diff
@@ -12,6 +12,7 @@
     "test:watch": "jest --watch"
   },
   "dependencies": {
+    "classnames": "^2.5.1",
```

Comment:
The Swiss Army knife of CSS class utilities. You can combine a wide variety of CSS class strings in a wide variety of ways - arrays, objects, multiple strings, conditional logic, etc. And `classnames` just figures things out.

https://www.npmjs.com/package/classnames